### ISSUE TITLE
[Messenger] Adapter options has goofy syntax

### ISSUE THREAD

**Issue by davidbarratt** [symfony/symfony#26936]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 4.1

If you want to set the name of the exchange, outside of the DSN, you have to use a syntax like this:
```yaml
framework:
    messenger:
        adapters:
          rsvp:
            dsn: "%env(MESSENGER_ADAPTER_DSN)%"
            options:
              -
                name: exchange
                value:
                  name: rsvp
```

Shouldn't this just be something like this?
```yaml
framework:
    messenger:
        adapters:
          rsvp:
            dsn: "%env(MESSENGER_ADAPTER_DSN)%"
            options:
              exchange:
                name: rsvp
```

**Comment by sroze** [comment]
Fair point. I'd 👍such syntax over the first one, definitely.

**Comment by sroze** [comment]
Should be fixed in #26947.

**Comment by sroze** [comment]
Fixed by #26947.
